libraries/c_sdk/standard/mqtt/include/core_mqtt_config.h

```diff
@@ -1,5 +1,5 @@
-#ifndef MQTT_CONFIG_H_
-#define MQTT_CONFIG_H_
+#ifndef CORE_MQTT_CONFIG_H_
```

Comment:
Can the config file be removed? We are keeping the config file only in the tests and demo projects. 
In this [PR](https://github.com/aws/amazon-freertos/pull/2482) for adding coreMQTT integration tests, I have added the config file to both the `aws_test` and `aws_demo` folders of the target board(s)

Reply:
Yea. I was thinking to create a separate PR for doing that. I knew that by removing interface from cmake, we will have to create core_mqtt_config.h in each demo/tests for vendor.

Reply:
Addressed the comments.